pombola/south_africa/templates/core/organisation_detail.html

```diff
@@ -39,6 +39,22 @@
     {% include "core/_position_listing.html" with positions=positions %}
   {% endif %}
 
+  <div class="pagination">
+    <span class="step-links">
```

Comment:
Having this extra `<span>` block inside the pagination container breaks the CSS button highlighting so the the whole pagination block rather than the target button lights up red on mouseover. Taking this out seems to fix it.